internal/cmd/cmd.go

```diff
@@ -20,7 +20,7 @@ type Command struct {
 	// "<Long>"
 	Long string
 	// Cmd is the function that is called when the command is matched.
-	Cmd func()
```

Comment:
i dont actually agree with this change.

This forces all returns to be an error and hides the formatting of the message. Before, each command could return whatever it needed in any way it needed. Now everything must be an error and you have less direct control of how its presented per command. All this does is sacrifice flexibility for a small bit of DRY for the error